contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+
+Spending only five minutes estimating something your team will spend weeks on doesn't make sense.
+
+### How to fix it 
+
+Once you have shortlisted the ideas you want to work on, the next step is to estimate the dollar impact of each one. Because you'll know **exactly** what to expect from your experiments, the result is a higher win rate.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a handy [sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
+
+Alexey says that one of their biggest wins came from following this approach:
+
+"At MasterClass, we had one-size-fits-all pricing. We knew we would eventually need to add tiered pricing, but this would be a huge, months-long company effort. We struggled to prioritize it."
+
+Enter EVELYN. Alexey knew they needed a dollar estimate of the impact.
+
+"Instead of building the entire backend, we created a simple [fake door test](https://posthog.com/tutorials/fake-door-test) on the frontend. Users could select their preferred tier – but at checkout, we let them know they were being upgraded to the best tier and at the cheapest price. In reality, this was actually the existing one-size-fits-all pricing.
+
+Now we knew how much money the company stood to make! We prioritized it immediately, and it was incredibly easy to ask the rest of the company to work on it for a few months."
+
+## Problem 3: Your experiment win size is low
+
+Your growth team has been around for a few quarters and you've picked all the low-hanging fruit. Your new experiments require more effort AND have less impact. Now what?
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Increase the number of high-quality project ideas.
+2. Maximize the impact of every experiment you ship.
+
+### Increase the number of high-quality project ideas
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
+
+### Maximize the impact of every experiment you ship
```

Comment:
Echoing my comment above, this feels weak. How do they improve the initial hypothesis? How do they know which ones will have an impact and which ones won't? If just shipping more iterations, that should be the advice.

Reply:
reworked